shared/templates/csv/packages_installed.csv

```diff
@@ -7,6 +7,7 @@ firewalld
 iptables
 libcap-ng-utils
 openscap-scanner
+openssh-server
```

Comment:
Do you think it can make a problem on SUSE systems where it's just `openssh`?

Reply:
I think it shouldn't make a problem because there are 2 different rules - first for SUSE and second for other operating systems.

Reply:
It would bloat up SUSE's content with useless check for `openssh-server` package.
I have moved the entry to `wrlinux`'es CSV files to avoid bloating up content.